README.md

```diff
@@ -0,0 +1,270 @@
+implemented and reviewed via the normal GitHub pull request workflow. For more
+information, please reference the [CONTRIBUTING][contributing] document.
+
+[contributing]: https://github.com/scipian/community/blob/master/CONTRIBUTING.md
+
+
+Table of Contents
+-----------------
+
+- [Thanks][thanks]
+- [When You Need to Follow This Process][when]
+- [Before Creating an RFC][before]
+- [Process][process]
+- [Life Cycle][life-cycle]
+- [Reviewing][reviewing]
+- [Implementing][implementing]
+- [Holding][holding]
+- [Governance][governance]
+- [License][license]
+- [Contributions][contributions]
+
+[thanks]: /README.md#thanks
+[when]: /README.md#when-you-need-to-follow-this-process
+[before]: /README.md#before-creating-an-rfc
+[process]: /README.md#process
+[life-cycle]: /README.md#life-cycle
+[reviewing]: /README.md#reviewing
+[implementing]: /README.md#implementing
+[holding]: /README.md#holding
+[governance]: /README.md#governance
+[license]: /README.md#license
+[contributions]: /README.md#contributions
+
+Thanks
+------
+
+First, before we begin, this process is heavily inspired by the 
+[Rust Community][rust-governance]. You are a great example of how a community 
+can operate and thrive. Thank you!
+
+[rust-governance]: https://www.rust-lang.org/governance
+
+When You Need to Follow This Process
+------------------------------------
+
+You need to follow this process if you intend to make `substantial` changes to
+the platform or the RFC process itself. The definition of a `substantial` change
+is evolving based on community norms and varies depending on what part of the
+ecosystem you are proposing to change.
+
+Here are some examples of when an RFC is required:
+
+- Any semantic change to the platform that is not a bugfix.
+- Removing features, including those that are feature-gated.
+- Creating roadmaps that encompass where the platform is going.
+
+Here are some examples of when an RFC is not required:
+
+- Rephrasing, reorganizing, refactoring, or otherwise "changing shape does not
+  change meaning".
+- Additions that strictly improve the code base like performance/speedup, better
+  error handling, and etc.).
+- Additions only likely to be noticed by other developers of scipian, invisible
+  to users of scipian.
+
+_DISCLAIMER:_ If you submit a pull request to implement a new feature without 
+going through the RFC process, it may be closed with a polite request to submit 
+an RFC first.
+
+Before Creating an RFC
+----------------------
+
+Please do not try to rush the process, there are many reasons it is in place and
+one major reason is having a well thought out feature that is documented. This
+will allow implementing the feature much smoother, because you will know what 
+you are building before any work is done.
+
+Rushing the process will definitely hurt it's chances of acceptance. For example
+RFCs that are hastily, low quality, previously rejected, or those that don't fit
```

Comment:
hastily written? hastily proposed? I think we need a verb for what is being done hastily to the RFC.

Reply:
I liked `proposed`.